docs/java/features/extensions/extension-library/curconv/currency-conversion.md

```diff
@@ -85,5 +241,21 @@ You can report a BCP incident or error through the SAP Support Portal Informatio
 
 ### Frequently asked questions
 
-- For [Currency Conversion library](troubleshooting-and-faq#currency-conversion)
-- For [Integration Objects](troubleshooting-and-faq#integration-objects)
+Q: Do you provide currency exchange rates from a data provider?
```

Comment:
Idea: Use callouts to better visualize the Q and A section: https://v2.docusaurus.io/docs/markdown-features/#calloutsadmonitions and to make it look a bit more appealing.
Currently it is a bit unreadable due to missing visual differentiation between Qs and As.